internal/server/manage_assurance_visits_test.go

```diff
@@ -202,3 +202,33 @@ func TestPostManagePDR(t *testing.T) {
 	assert.Equal(http.StatusOK, resp.StatusCode)
 	assert.Equal(Redirect("/123/assurance-visits?success=managePDR"), redirect)
 }
+
+func TestParseVisitForm(t *testing.T) {
+	assert := assert.New(t)
+
+	visitForm := sirius.AssuranceVisitDetails{
+		CommissionedDate:    "2020-01-01",
+		ReportDueDate:       "2020-01-02",
+		ReportReceivedDate:  "2020-01-03",
+		VisitOutcome:        "Cancelled",
+		PdrOutcome:          "Cancelled",
+		ReportReviewDate:    "2020-01-04",
+		VisitReportMarkedAs: "Successful",
+		VisitorAllocated:    "John Johnson",
+		ReviewedBy:          1,
+		Note:                "Test notes",
+	}
+
+	assert.Equal(parseVisitForm(visitForm), sirius.AssuranceVisit{
```

Comment:
doesn't _really_ matter that much, but "expected" should be the first argument, followed by "actual"